### PR TITLE
chore: explicitly fetch pull request labels

### PR DESCRIPTION
**Pull request by frezbo**
Fetch the PR labels explicitly and not for
`github.event.pull_request.labels`. This means any labels added after the pull request have been created are available to a job re-run.

### CODE REVIEW

**Review by smira — COMMENTED** [review]
no changes for this repo github workflows?

**Comment by frezbo** [comment]
> no changes for this repo github workflows?

yes, this is for customsteps only, I'll get it tested with omni before merging

**Review by smira — APPROVED** [review]

File: internal/config/constants.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ const (
 	GoImportsVersion = "v0.14.0"
 	// GolangCIlintVersion is the version of golangci-lint.
 	// renovate: datasource=go depName=github.com/golangci/golangci-lint
-	GolangCIlintVersion = "v1.55.0"
+	GolangCIlintVersion = "v1.55.1"
```
[the remark this answers is not in the document]
Comment (frezbo) [reply internal/config/constants.go:43]: done

**Review by smira — APPROVED** [review]

**Pull request merged**